README.md

```diff
@@ -85,9 +79,9 @@ go get -u -v github.com/btcsuite/btcwallet/...
 
 ## Getting Started
 
-The following instructions detail how to get started with btcwallet
-connecting to a localhost btcd.  Commands should be run in `cmd.exe`
-or PowerShell on Windows, or any terminal emulator on *nix.
+The following instructions detail how to get started with btcwallet connecting
+to a localhost testnet btcd.  Commands should be run in `cmd.exe` or PowerShell
```

Comment:
Makes sense.  I would actually like to see the documentation improved quite a bit to urge Windows users towards Paymetheus, but that's out of scope for this PR.